Makefile

```diff
@@ -98,6 +98,10 @@ kill-anvil: ## kill process running at port 8545 (default port of anvil)
 localcluster: ## spin up the localcluster using the default configuration file
 	@python -m sdk.python.localcluster --config ./sdk/python/localcluster.params.yml --fully_connected
 
+.PHONY: localcluster-expose1
+localcluster-expose1: ## spin up the localcluster exposing node 1 API
```

Comment:
Seems oddly specific.

Reply:
I mean, exposing only node 1 of the entire cluster?

Reply:
That's exactly what it does.